CHANGELOG.md

```diff
@@ -8,9 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- New filter functions to repair a smiles that mismatches parent mass
```

Comment:
Expand a bit. 
- Mention PR --> ` [#440](https://github.com/matchms/matchms/pull/440)`
- Make it multiple points to name all things that now can be done (correcting parent mass, precursor mz, adducts...) but also to remove spectra with inconsistencies between precursor and annotated smiles.